hnn_core/network.py

```diff
@@ -285,10 +284,10 @@ def _gid_assign(self):
                 pc.set_gid2node(gid_input, rank)
                 self._gid_list.append(gid_input)
         # legacy handling of the external inputs
-        # NOT perfectly balanced for now
-        for gid_base in range(rank, self.N_extinput, nhosts):
-            # shift the gid_base to the extinput gid
-            gid = gid_base + self.gid_dict['extinput'][0]
+        # NOT perfectly balanced for now  XXX wat?
```

Comment:
just delete this comment if it makes no sense. We can always look at HNN or check an older commit. I guess it's helpful to nobody to have comments that make no sense.